module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BasicSearchTest.php

```diff
@@ -53,61 +55,105 @@ public function testOutOfBoundsPage()
         $session->visit($baseUrl . '&page=1000');
         $this->assertEquals($baseUrl . '&page=1', $session->getCurrentUrl());
         $page = $session->getPage();
-        $this->assertMatchesRegularExpression(
-            "/Showing 1 - 1 results of 1 for search 'id:testbug1', query time: .*/",
+        $this->assertStringStartsWith(
+            'Showing 1 - 1 results of 1',
             trim($this->findCss($page, '.search-stats')->getText())
         );
     }
 
     /**
-     * Test simple top pagination
+     * Data provider for testDefaultTopPagination
+     *
+     * @return array
+     */
+    public function topPaginationProvider(): array
+    {
+        return [
+            [false],
+            [true],
+        ];
+    }
+
+    /**
+     * Test default top pagination
+     *
+     * @param bool $jsResults Whether to update search results with JS
+     *
+     * @dataProvider topPaginationProvider
      *
      * @return void
      */
-    public function testSimpleTopPagination(): void
+    public function testDefaultTopPagination(bool $jsResults): void
     {
+        // Change configuration:
+        $this->changeConfigs(
+            [
+                'searches' => [
+                    'General' => [
+                        'load_results_with_js' => $jsResults,
+                    ],
+                ],
+            ]
+        );
+
         $session = $this->getMinkSession();
         $session->visit($this->getVuFindUrl() . '/Search/Results');
         $page = $session->getPage();
 
-        // No paginator unless configured:
-        $this->unFindCss($page, '.search-header .pagination');
-        $this->unFindCss($page, '.search-header .pagination-simple');
+        // Should never have full pagination:
```

Comment:
Should this be "never have full top pagination"?

Reply:
Yes, fixed.